src/main/java/mate/academy/web/filter/AuthenticationFilter.java

```diff
@@ -0,0 +1,51 @@
+package mate.academy.web.filter;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.lib.Injector;
+import mate.academy.service.DriverService;
+
+public class AuthenticationFilter implements Filter {
+    private static final String DRIVER_ID = "driver_id";
+    private static final Injector injector = Injector.getInstance("mate.academy");
+    private static final DriverService driverService =
```

Comment:
do we need these lines?

Reply:
Sorry, now there is no need, because check of it was removed from if condition... ^__^
Changed.